wagtail/wagtailcore/management/commands/set_url_paths.py

```diff
@@ -1,17 +1,20 @@
 from __future__ import absolute_import, unicode_literals
 
-from django.core.management.base import NoArgsCommand
-
+from django.core.management.base import BaseCommand
 from wagtail.wagtailcore.models import Page
 
 
-class Command(NoArgsCommand):
+class Command(BaseCommand):
+
+    help = 'Resets url_path fields on each page recursively'
+
     def set_subtree(self, root, root_path):
         root.url_path = root_path
         root.save(update_fields=['url_path'])
         for child in root.get_children():
             self.set_subtree(child, root_path + child.slug + '/')
 
-    def handle_noargs(self, **options):
+    def handle(self, *args, **options):
         for node in Page.get_root_nodes():
             self.set_subtree(node, '/')
+
```

Comment:
flake8 complains about this blank line (http://46.101.88.142/torchbox/wagtail/605), so I guess it should not be there ;-)